src/vs/platform/native/common/native.ts

```diff
@@ -72,6 +72,8 @@ export interface ICommonNativeHostService {
 
 	setMinimumSize(width: number | undefined, height: number | undefined): Promise<void>;
 
+	saveWindowSplash(splash: IPartsSplash): Promise<void>;
```

Comment:
This is maybe not the best service to put it, ideally we had a theme service that the window can talk to that travels to the main side. We can think about introducing a new service for this if we wanted to.